src/object/obj_rpc.h

```diff
@@ -31,7 +31,7 @@
  * These are for daos_rpc::dr_opc and DAOS_RPC_OPCODE(opc, ...) rather than
  * crt_req_create(..., opc, ...). See daos_rpc.h.
  */
-#define DAOS_OBJ_VERSION 8
+#define DAOS_OBJ_VERSION 9
```

Comment:
Fan Yong mentioned that in release2.2, this version is 9, to keep inter between 2.2 and 2.4, should this bump to 10?

Reply:
The version is 2.2.0 was 8, in 2.2.1 candidate is 9. We prepare to change it back to  8 in 2.2.1 release.

Reply:
yes already merged the bump back in 2.2 to 8.
note that there is not going to be a 2.2.1 release. so need not to worry about that.